src/besselk.jl

```diff
@@ -431,3 +434,68 @@ end
 besselk_power_series_cutoff(nu, x::Float64) = x < 2.0 || nu > 1.6x - 1.0
 besselk_power_series_cutoff(nu, x::Float32) = x < 10.0f0 || nu > 1.65f0*x - 8.0f0
 
+
+"""
+  besselk_asymptoticexp(v, x, order)
+
+Computes the asymptotic expansion of K_ν w.r.t. argument. Accurate for large x, and faster than uniform asymptotic expansion for small to small-ish orders. The default order of the expansion is 10.
+"""
+function besselk_asymptoticexp(v, x::T) where T 
+  _besselk_asymptoticexp(v, float(x), 10)
+end
+besselk_asexp_cutoff(nu, x) = (nu < 20.0) && (x > 25.0)
```

Comment:
Should we have different cutoffs for Float64 and Float32?

Reply:
I thought about that, but I don't actually have much of an intuition for what ways they should be different. I'll play around with it a bit.

Reply:
I added a const `ASEXP_CUTOFF` in the consts file. For now they're both `35`, which is what I thought I had typed in this function actually. But I'm sort of still thinking of that value as a placeholder.